src/js/components/Image/Image.js

```diff
@@ -1,10 +1,22 @@
+import React, { useState } from 'react';
 import { StyledImage } from './StyledImage';
 
+const Image = ({ src, fallback, ...rest }) => {
+  const [imageMissing, setImageMissing] = useState(false);
+  return (
+    <StyledImage
+      {...rest}
+      onError={() => setImageMissing(true)}
+      src={!imageMissing ? src : fallback}
```

Comment:
I'm thinking it would be nice to see a warning on the console when fallback is applied. thoughts?

Reply:
hm, since the fallback image is an opt-in method i would say it is not needed.